common/beerocks/bwl/dummy/nl80211_client_dummy.cpp

```diff
@@ -110,6 +110,24 @@ bool nl80211_client_dummy::get_sta_info(const std::string &interface_name,
     return true;
 }
 
+bool nl80211_client_dummy::get_survey_info(const std::string &interface_name,
+                                           std::vector<survey_info> &survey_info_list)
+{
+    for (size_t i = 0; i < 8; i++) {
+        survey_info survey_info;
+
+        survey_info.in_use        = (0 == i);
+        survey_info.frequency_mhz = 2400 + i * 10;
```

Comment:
I think we should at least use valid frequencies...
```suggestion
        survey_info.frequency_mhz = 2412 + i * 5;
```

Reply:
Done